src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 0.1);
+}
+
+header .logo {
+  display: inline;
+  float: left;
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+}
+
+.nav,
+.link__box {
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+}
+
+.link__box {
+  padding-left: 0;
+  width: 100%;
+}
+
+.link__item {
+  display: inline-block;
+  align-items: center;
+}
+
+.link__elem {
+  display: flex;
+  align-items: center;
+  position: relative;
+  height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  margin-left: 20px;
+}
+
+.is-active {
+  margin-left: 0;
```

Comment:
you should remove margin from the first element, not the active one. what if you'd put `is-active` on 5th element?

Reply:
done